wagtail/admin/tests/test_menu.py

```diff
@@ -54,6 +56,34 @@ def test_simple_menu(self):
             ],
         )
 
+    def test_model_admin_add_to_admin(self):
+        menu = Menu(register_hook_name="register_menu_item")
+
+        # Only the friend admin should show, enemy admin is excluded from the menu.
+        with hooks.register_temporarily(
```

Comment:
I don't think we want this bit here. The aim of the test here is to verify that the modeladmin_register line in wagtail_hooks does or doesn't register the menu item, depending on the `add_to_admin_menu` setting. Here we're forcibly registering the menu items ourselves, so that doesn't get tested.

We probably just want a test that fetches the admin homepage (like [this one](https://github.com/wagtail/wagtail/blob/8e4a4fae5df1d77f752f4a476783bf81804e490e/wagtail/admin/tests/tests.py#L34-L55)), and checks that the link to `/admin/friendadmin/` is present on the page but `/admin/enemyadmin/` (is that the expected URL? Might need a `base_url_path = "enemyadmin"` line for consistency) isn't.